src/webgpu/util/conversion.ts

```diff
@@ -779,6 +779,8 @@ export class Scalar {
     };
     if (isFinite(this.value as number)) {
       switch (this.type.kind) {
+        case 'f64':
```

Comment:
I'm wondering whether we should have a different type for `abstract-float`. While they are currently represented as `f64`, they are special - no suffix, unique rules around concretization, untypable, etc.
The day WGSL gets a proper `f64` type, or `abstract-float`'s precision gets bumped to something > 64 bits, it's going to be hard to distinguish `f64` from `abstract-float`.

Reply:
Done